candle-wasm-examples/segment-anything/lib-example.html

```diff
@@ -163,6 +169,45 @@
       undoBtn.addEventListener("click", () => {
         undoPoint();
       });
+      // add event to download btn
+      downloadBtn.addEventListener("click", async () => {
+        // Function to load image blobs as Image elements asynchronously
+        const loadImageAsync = (imageURL) => {
+          return new Promise((resolve) => {
+            const img = new Image();
+            img.onload = () => {
+              resolve(img);
+            };
```

Comment:
can you please add this here? I was getting tainted canvas with the example imagas

```
img.crossOrigin = "anonymous";
```